keps/sig-ui/5008-kep-template/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Kubernetes Desktop
+kep-number: 5008
+authors:
+  - "@joaquimrocha"
+owning-sig: sig-ui
+participating-sigs:
```

Comment:
Maybe SIG CLI too?

Reply:
Thank. We had discussed this KEP with the sig-ui. But they may decide to include others as needed.